server/database/queries/getUserHome.js

```diff
@@ -0,0 +1,16 @@
+const dbconniction = require('../db_connection')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+const userHome = (callBack) => {
+  const sql = {
+    text: 'SELECT * FROM events WHERE user_id =$1',
+    values: [11]
```

Comment:
What is the values here?

Reply:
here the values  is the user_id .... we will change it when we build the cookies